Cache/Redis.php

```diff
@@ -4,6 +4,7 @@
 
 use Handlebars\Cache;
 use Predis\Client as Predis;
+use Redis;
```

Comment:
oh well. phpunit reveils i did not think enough. what about RedisClient and PredisClient?